tools/linter_lib/custom_check.py

```diff
@@ -317,8 +317,8 @@
         },
         {
             "pattern": r"""\Wjson_error\(['"].+[),]$""",
-            "exclude": {"zerver/tests"},
-            "description": "Argument to json_error should a literal string enclosed by _()",
+            "exclude": {"zerver/tests", "zerver/views/development/"},
+            "description": "Argument to json_error should be a literal string enclosed by _()",
```

Comment:
I merged this commit as 3da6f3c0afe92d0d5e5584076518a482fcb6a5a8, after a small commit message tweak.